src/golang/lib/models/schema_version.go

```diff
@@ -9,7 +9,7 @@ const (
 	// This is the source of truth for the required schema version
 	// for both the server and executor. This value MUST be updated
 	// when a new schema change is added.
-	CurrentSchemaVersion = 22
+	CurrentSchemaVersion = 23
```

Comment:
@kenxu95 not changing this caused data integration test failure as the schema is not updated, thus not matching some updated queries